.github/workflows/build.yml

```
@@ -13,7 +13,11 @@
 #
 
 name: Build
-on: [ push, pull_request ]
+  # Trigger the workflow on pushes to only the 'main' branch (this avoids duplicate checks being run e.g. for dependabot pull requests)
+  push:
+    branches: [ 2022 ] # TODO: Update if branching strategy changes
+  # Trigger the workflow on any pull request
+  pull_request:
```

not sure if i have indentation correct, but you may need to add this
```suggestion
  # Trigger the workflow on any pull request
  pull_request:
    types: [assigned, opened, synchronize, reopened]
```